src/App/routes/user.js

```diff
@@ -0,0 +1,8 @@
+import { Router } from 'express';
+import User from '../controllers/user';
+
+const router = Router();
+
+router.route('/login').post(User.login);
```

Comment:
Move this route to auth Routes please!! same for the controllers.

Reply:
Okay

Reply:
This one is users related route and I haven't handled them well yet!